src/app/view.component.ts

```diff
@@ -234,8 +234,10 @@ export class ViewComponent implements OnInit {
     }
     let r: any[];
     r = str.split('|');
-    const newTitle = r[r.length - 2] + ' | ' + r[r.length - 1];
-    return newTitle;
+    if (r.length < 5) {
```

Comment:
Here it is! Also, I'm not a fan of this length hack. Let's keep it for now, but is there a particular naming scheme you want to always do? We could modify the database to return one extra column that has the name you want. This is the file ( https://github.com/dafrenchyman/mrsharky/blob/master/process/Docker/files/Datasets.ods ) that we use to populate a CSV that we then load into the database. Can you come up with names a new column that has the names you want to display for each of the datasets in here?